dotCMS/src/main/java/com/dotmarketing/util/WebKeys.java

```diff
@@ -258,9 +258,14 @@
     public static final String ADMIN_MODE_COOKIE = "ADMIN_MODE_COOKIE";
 
     // SESSION ATTRIBUTES
+    /*
     public static final String EDIT_MODE_SESSION = "com.dotmarketing.EDIT_MODE_SESSION";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)